### ISSUE TITLE
Creating attributes causes changes to existing attributes

### ISSUE THREAD

**Issue by hemalshah-gradientedge** [labd/terraform-provider-commercetools#262]
When adding or removing new product type attributes, existing attributes get amended which have not changed.

Here is an example of attributes that have not changed.
```
~attribute {
~constraint = "None" ->"SameForAll"
~label      = {
~"en" = "Length" ->"Weight Unit of Measure"
            }
~name       = "length" ->"weightUom"
```

The issue here is that the constraint cant change from None to SameForAll which leads to the error:
```
Error: At the moment it is only possible to change the attribute constraint from `SameForAll` and `Unique` to `None`.
```

**Comment by onur9** [comment]
Hello @hemalshah-gradientedge 

Thank you for reporting. Can you include a full example?

**Comment by mvantellingen** [comment]
This seems to be fixed in the 1.3.0 (to be released). Let us know if that is not the case
